tile.go

```diff
@@ -28,9 +43,10 @@ func (t *Tile) Deg2Num() (x, y int, err error) {
 
 	// Check that input lat/long values are within WGS84 bounds
 	if t.Lat < -85.0511 || t.Lat > 85.0511 || t.Long < -180.0 || t.Long > 180.0 {
-		err := fmt.Errorf("one or both outside valid range (Long, Lat): (%v, %v)", t.Long, t.Lat)
-		log.Print(err)
-		return -1, -1, err
+		return 0, 0, ErrInvalidLatLong{
```

Comment:
I suggest returning values that are obviously invalid.  There's no way to guarantee that the user of this will check the return value.  Values that are obviously invalid will help locate those cases.

Reply:
I think we should assume that the end user always handles errors. This is expected in Go (https://github.com/golang/go/wiki/CodeReviewComments#handle-errors), so handing back anything but the 0 value to me seems strange. I was going to ask you about those values and if they had significance. Thanks for chiming in on that change.